modules/system/classes/ImageResizer.php

```diff
@@ -0,0 +1,768 @@
+<?php namespace System\Classes;
+
+use Url;
+use Crypt;
+use Cache;
+use Event;
+use Config;
+use Storage;
+use Exception;
+use SystemException;
+use File as FileHelper;
+use System\Models\File as SystemFileModel;
+use October\Rain\Database\Attach\File as FileModel;
+use October\Rain\Database\Attach\Resizer as DefaultResizer;
+
+/**
+ * $width = numeric, 'auto' | false | null
+ * $height = numeric, 'auto' | false | null
```

Comment:
@LukeTowers your docs here state that width and height can be specified as a string `auto`, however, I don't see any handling of that value. The docblock for the constructor says only integer, bool and null are accepted for those, and they are converted to an int on construction.

Reply:
@bennothommo oversight on my part, just need to make sure that the constructor converts 'auto' to 0.

Reply:
@LukeTowers no worries - I've added that in for you.